azurerm/internal/services/keyvault/registration.go

```diff
@@ -35,12 +35,14 @@ func (r Registration) SupportedDataSources() map[string]*schema.Resource {
 // SupportedResources returns the supported Resources supported by this Service
 func (r Registration) SupportedResources() map[string]*schema.Resource {
 	return map[string]*schema.Resource{
-		"azurerm_key_vault_access_policy":                    resourceKeyVaultAccessPolicy(),
-		"azurerm_key_vault_certificate":                      resourceKeyVaultCertificate(),
-		"azurerm_key_vault_certificate_issuer":               resourceKeyVaultCertificateIssuer(),
-		"azurerm_key_vault_key":                              resourceKeyVaultKey(),
-		"azurerm_key_vault_managed_hardware_security_module": resourceKeyVaultManagedHardwareSecurityModule(),
-		"azurerm_key_vault_secret":                           resourceKeyVaultSecret(),
-		"azurerm_key_vault":                                  resourceKeyVault(),
+		"azurerm_key_vault_access_policy":                         resourceKeyVaultAccessPolicy(),
+		"azurerm_key_vault_certificate":                           resourceKeyVaultCertificate(),
+		"azurerm_key_vault_certificate_issuer":                    resourceKeyVaultCertificateIssuer(),
+		"azurerm_key_vault_key":                                   resourceKeyVaultKey(),
+		"azurerm_key_vault_managed_hardware_security_module":      resourceKeyVaultManagedHardwareSecurityModule(),
+		"azurerm_key_vault_secret":                                resourceKeyVaultSecret(),
+		"azurerm_key_vault":                                       resourceKeyVault(),
+		"azurerm_key_vault_managed_storage_account":               resourceKeyVaultManagedStorageAccount(),
+		"azurerm_key_vault_managed_storage_account_sasdefinition": resourceKeyVaultManagedStorageAccountSasToken(),
```

Comment:
this name seems incomplete, should we make it?
```suggestion
		"azurerm_key_vault_managed_storage_account_sas_token_definition": resourceKeyVaultManagedStorageAccountSasTokenDefinition(),
```